pkg/securityscan/core/configmap.go

```diff
@@ -43,15 +45,19 @@ func NewConfigMaps(clusterscan *cisoperatorapiv1.ClusterScan, clusterscanprofile
 	}
 	cmMap["configcm"] = configcm
 
+	logrus.Infof("CustomBenchmarkConfigMapName Set: %v %v", clusterscanbenchmark.Spec.CustomBenchmarkConfigMapName, clusterscanbenchmark.Spec.CustomBenchmarkConfigMapNamespace)
```

Comment:
These logs have been removed in the last commit in this PR, were added for debugging. Will squash all commits before merge.